utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -653,6 +665,98 @@ int main(int argc, char **argv)
+
+                        idx++;
+                        anchorBoxInfoTensor[idx].lt.x = 0.75 * roiTensorPtrSrc[i].xywhROI.roiWidth;
+                        anchorBoxInfoTensor[idx].lt.y = 0.125 * roiTensorPtrSrc[i].xywhROI.roiHeight;
+                        anchorBoxInfoTensor[idx].rb.x = 0.875 * roiTensorPtrSrc[i].xywhROI.roiWidth;
+                        anchorBoxInfoTensor[idx].rb.y = 0.5 * roiTensorPtrSrc[i].xywhROI.roiHeight;
+
+                        if(srcDescPtr->c == 3)
+                        {
+                            int idx = boxesInEachImage * 3 * i;
+                            colorBuffer[idx] = 0;
+                            colorBuffer[idx + 1] = 0;
+                            colorBuffer[idx + 2] = 240;
+                            colorBuffer[idx + 3] = 0;
+                            colorBuffer[idx + 4] = 240;
+                            colorBuffer[idx + 5] = 0;
+                            colorBuffer[idx + 6] = 240;
+                            colorBuffer[idx + 7] = 0;
+                            colorBuffer[idx + 8] = 0;
+
+                            for (int j = 0; j < 9; j++)
+                            {
+                                if (!inputBitDepth)
+                                    colors8u[idx + j] = (Rpp8u)(colorBuffer[idx + j]);
+                                else if (inputBitDepth == 1)
+                                    colors16f[idx + j] = (Rpp16f)(colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 2)
+                                    colors32f[idx + j] = (Rpp32f)(colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 5)
+                                    colors8s[idx + j] = (Rpp8s)(colorBuffer[idx + j] - 128);
+                            }
+                        }
+                        else
+                        {
+                            int idx = boxesInEachImage * i;
+                            colorBuffer[idx] = 240;
+                            colorBuffer[idx + 1] = 120;
+                            colorBuffer[idx + 2] = 60;
+
+                            for (int j = 0; j < 3; j++)
+                            {
+                                if (!inputBitDepth)
+                                    colors8u[idx + j] = (Rpp8u)(colorBuffer[idx + j]);
+                                else if (inputBitDepth == 1)
+                                    colors16f[idx + j] = (Rpp16f)(
+                                        colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 2)
+                                    colors32f[idx + j] = (Rpp32f)(colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 5)
+                                    colors8s[idx + j] = (Rpp8s)(colorBuffer[idx + j] - 128);
+                            }
+                        }
+                    }
```

Comment:
The initialization looks long for this function
can we add a init_erase in common header similar to how it is added for RICAP?

Reply:
Added it.